src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/CompilationTask.kt

```diff
@@ -198,6 +198,9 @@ internal fun JvmCompilationTask.runPlugins(
           .plus(
             kaptArgs(context, plugins, "stubsAndApt"),
           )
+          .plus(
+            CompilationArgs(info.passthroughFlagsList),
```

Comment:
I'm a little afraid of this one. Need to go look at the docs and make sure it can't get into arguments.